src/views/EdgeApplications/FormFields/FormFieldsCreateEdgeApplications.vue

```diff
@@ -428,10 +434,10 @@
           v-model="hostHeader"
           :class="{ 'p-invalid': hostHeaderError }"
           aria-describedby="hostHeader-help"
+          placeholder="${host}"
         />
         <div class="text-color-secondary text-sm font-normal">
-          Allow the origin to identify the virtualhost and locate the content or application. If
-          this is blank, Azion will use the Address as the default.
+          Identify a virtualhost sent in the Host header to the origin.
```

Comment:
acho bom deixar essa parte: If blank, Azion will use the Address as the default.
Dá um contexto.

Reply:
Não tem como ficar blank, é required :/

Reply:
Perfeito!